Update enterprise licensing documentation (#16615)
hashicorp/nomad: website/content/docs/enterprise/license/faq.mdx

# Frequently Asked Questions (FAQ)

This FAQ is for the license changes introduced in Nomad Enterprise version v1.6.0+ent.

For license-related FAQ for earlier versions, please refer to earlier versions of this document
using the version drop-down on this page, e.g. [v1.5.x](/nomad/docs/v1.5.x/enterprise/license/faq).

## Q: Can I get a quick summary of the Nomad changes?

Starting with Nomad Enterprise v1.6.0, license expiration is different.

For trial licenses, the behavior remains the same.  Enterprise features will stop working

> when the license expires, and a server can not start with an expired license.

Reviewer: ```suggestion
when the license expires, and a server cannot start with an expired license.
```

I think? How does English even work? 😅

Author: looks like grammarly agrees with you https://www.grammarly.com/blog/cannot-or-can-not/ but to me it's a little overly prescriptive.

I personally think they're interchangeable but that the extra space adds emphasis, but the important bit is the _"not"_ part, so as long as that's unambiguous I could roll with "cannot" 🤷 

Tangentially, in creative writing I tend to read "cannot" as a little, um, posh or hoity-toity?  I suppose because ordinary folks usually say "can't" unless they want to emphasize "can _not"_ which benefits from the space to indicate a short pause.  I could go extra informal like

> and a server _literally **can't even**_ start with an expired license.  🤣 🤣 🤣

Reviewer: Ah interesting! Don't mind me then, I think I never thought about "can not" so it seemed a bit odd (can? not!) 😅

For new "non-terminating" contract licenses, instead of expiration being compared to the current time,

> expiration time is compared to the build date of the Nomad binary, which you can see with `nomad version`.

Reviewer: ```suggestion
expiration time is compared to the build date of the Nomad binary, which you can find in the output of the [`nomad version`](<insert URL here>) command.
```

> The practical result is that newer contract licenses will work in perpetuity for any v1.6+ version of Nomad

Reviewer: ```suggestion
The practical result is that newer contract licenses will work in perpetuity for any v1.6.0+ version of Nomad
```

There are some others below as well.

Every time `nomad agent` runs as a server (or in `-dev`), it requires a license.

> ## Q: May I downgrade Nomad Enterprise from v1.6 to an earlier version?

Reviewer: ```suggestion
## Q: May I downgrade Nomad Enterprise from v1.6.0 to an earlier version?
```

Missed one 😄
